pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -136,6 +159,79 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	// Don't modify the informer's copy.
 	tr := original.DeepCopy()
 
+	// build creation and status update adapted from prow: https://github.com/kubernetes/test-infra/blob/8faa040989cf8552683dcdd4400505ca62a3c408/prow/cmd/build/controller.go#L207
```

Comment:
There is a tracker defined in the reconciler (meant to track build status) that I don't see used

Reply:
I think I understand now, a tracker is used to monitor child object (in this case a Build) and when the tracker sees a change on the tracked object, it runs `reconcile`?  I have added a tracker.Track statement to reconcile, thanks for pointing out the tracker!

Reply:
@pivotal-nader-ziada 
could you clarify how you thought the tracker would be used, from the comment:
```
	// TODO use build tracker to fetch build
```
it seems that it might be possible to fetch a build with a tracker.  My understanding is that a tracker is an event watcher that runs `reconcile`.  Can you shed some light here?  Thanks!

Reply:
You are correct Aaron, the tracker is to watch the build. Sorry for the confusing comment